SDL/Event.cu

```diff
@@ -2094,13 +2096,13 @@ SDL::miniDoublets* SDL::Event::getMiniDoublets()
     if(mdsInCPU == nullptr)
     {
         mdsInCPU = new SDL::miniDoublets;
-        mdsInCPU->nMDs = new unsigned int[nLowerModules+1];
+        mdsInCPU->nMDs = new int[nLowerModules+1];
 
         //compute memory locations
         mdsInCPU->nMemoryLocations = new unsigned int;
         cudaMemcpyAsync(mdsInCPU->nMemoryLocations, mdsInGPU->nMemoryLocations, sizeof(unsigned int), cudaMemcpyDeviceToHost, stream);
         cudaStreamSynchronize(stream);
-        mdsInCPU->totOccupancyMDs = new unsigned int[nLowerModules+1];
+        mdsInCPU->totOccupancyMDs = new int[nLowerModules+1];
```

Comment:
The reason is because this is used as input to one of the Alpaka functions in the Minidoublets kernel, but that function does not accept unsigned int as an input. It only accepts int, so I had to change the type.